modules/gitops-additional-steps-for-disconnected-clusters.adoc

```diff
@@ -0,0 +1,43 @@
+[id="additional-steps-disconnected-clusters_{context}"]
+= Additional steps for disconnected clusters
+
+In a disconnected cluster, Keycloak communicates with OpenShift Oauth Server through proxy. Follow the additional steps to get Keycloak integrated with OpenShift Oauth Login.
+
+.Procedure
+
+. Login to the Keycloak Pod.
```

Comment:
Typically, in OpenShift documentation, if there's a code snippet after a command, the command ends with a colon, not a period. So it'd be `Login to the Keycloak Pod:` <next line, code snippet>. 

You are consistent with putting periods at the end of your commands, so I don't think it's a big deal, but if you have time, it'd be best to end a command with a colon if you have a code snippet or anything for users to copy right after the command.